server/src/main/java/org/apache/lucene/queries/BinaryDocValuesRangeQuery.java

```diff
@@ -145,25 +145,31 @@ public int hashCode() {
         INTERSECTS {
             @Override
             boolean matches(BytesRef from, BytesRef to, BytesRef otherFrom, BytesRef otherTo) {
-                // part of the other range must touch this range
-                // this:    |---------------|
-                // other:               |------|
+                /*
```

Comment:
Why is this converted to a multiline comment? Is there some problem with using several single line comments with the formatter?

Reply:
I observed the formatted mangling these diagrams, hence the change. You can ask the formatter to process block and javadoc comments, but I decided that wasn't worth it.